dev/tasks/linux-packages/apt/build.sh

```diff
@@ -68,6 +68,14 @@ else
   run cp -rp "/host/tmp/debian" debian
 fi
 # export DEB_BUILD_OPTIONS=noopt
+case "${VERSION}" in
+  *~dev*)
+    run dch -v "${VERSION}-1" "dev build"
+    ;;
+  *~rc*)
+    run dch -v "${VERSION}-1" "RC build"
+    ;;
+esac
```

Comment:
We update `debian/changelog` by `rake version:update` in `dev/tasks/linux-pakcages`.
So this is needless.

Reply:
Thanks. Will drop that part then.